CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+## [0.2.1] - 2023/02/20
+
+### Changed
+
+- micropip now depends on packaging>=0.23.0
```

Comment:
It was actually 23.0, not 0.23 but we can fix it some other time we modify the changelog.

Reply:
I think I confused with Pyodide version 😅